src/FtlServer.cpp

```diff
@@ -554,18 +548,21 @@ void FtlServer::eventStreamIdAssigned(std::shared_ptr<FtlServerStreamIdAssignedE
         return;
     }
     uint16_t mediaPort = portResult.Value;
+    std::shared_ptr<RtpPacketSink> rtpPacketSink = event->PacketSink;
 
     // Attempt to fire up the new FtlStream. Queue a new event when we're done.
     dispatchAsyncCall(
-        [this, event, control = std::move(control), mediaPort]() mutable
+        [this, event, control = std::move(control), mediaPort, rtpPacketSink]() mutable
         {
             std::unique_ptr<ConnectionTransport> mediaTransport = 
                 mediaConnectionCreator->CreateConnection(mediaPort, event->TargetAddr);
             auto stream = std::make_shared<FtlStream>(
                 std::move(control), std::move(mediaTransport), event->Metadata, event->StreamId,
                 std::bind(&FtlServer::onStreamClosed, this, std::placeholders::_1),
-                std::bind(&FtlServer::onStreamRtpPacket, this, std::placeholders::_1,
-                    std::placeholders::_2, std::placeholders::_3));
+                [rtpPacketSink](ftl_channel_id_t channelId, ftl_stream_id_t steamId, const std::vector<std::byte> packet)
```

Comment:
nit: Trying to keep lines <= 100 char

Reply:
Is there a way to auto format?

I can certainly manually format though